### DISCUSSION TITLE
Remove modules section in README

### DISCUSSION THREAD

**Discussion by moorereason** [pelletier/go-toml#815] [Ideas]
Since go-toml only supports the two most recent minor releases of Go, should we drop the **Modules** section of the README?  Modules were introduced in Go 1.13 (Sep 2019) and have been the standard since Go 1.16 (Feb 2021).

**Comment by pelletier** [comment]
I guess it's been a year since https://github.com/pelletier/go-toml/discussions/562. It's probably time to assume people who really don't know how to handle modules with go should start somewhere else. 👍🏻
